Framework/HistogramData/test/ScalableTest.h

```diff
@@ -60,6 +78,14 @@ class ScalableTest : public CxxTest::TestSuite {
     TS_ASSERT_DELTA(lhs[1], 0.6, 1e-14);
   }
 
+  void test_divide_reverse_order() {
```

Comment:
I would say we shouldn't allow this is it's going to be wrong on certain cases. If it's there it will be assumed it does the correct thing.